### PR TITLE
[luci] Support NEG operator in lang

### PR DESCRIPTION
**Pull request by kvochko**
This enables NEG operator in luci lang

For #1034 
Draft #1049 

ONE-DCO-1.0-Signed-off-by: Andrey Kvochko <a.kvochko@samsung.com>

### CODE REVIEW

File: compiler/luci/lang/include/luci/IR/Nodes/CircleNeg.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+namespace luci
+{
+
+/**
+ * @brief NEG in Circle
+ */
+class CircleNeg final : public FixedArityNode<1, CircleNodeImpl<CircleOpcode::NEG>>
+{
+public:
+  loco::Node *x(void) const { return at(0)->node(); }
```
Comment (ilya-palachev) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleNeg.h:34]:
```suggestion
  loco::Node *x() const { return at(0)->node(); }
```

File: compiler/luci/lang/include/luci/IR/Nodes/CircleNeg.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+namespace luci
+{
+
+/**
+ * @brief NEG in Circle
+ */
+class CircleNeg final : public FixedArityNode<1, CircleNodeImpl<CircleOpcode::NEG>>
+{
+public:
+  loco::Node *x() const { return at(0)->node(); }
```
Comment (seanshpark) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleNeg.h:34]: All other IRs have `(void)`. Please follow like others.

Comment (kvochko) [reply compiler/luci/lang/include/luci/IR/Nodes/CircleNeg.h]: Done!

**Comment by seanshpark** [comment]
@ilya-palachev , PTAL

**Pull request merged**